cime_config/buildnml

```diff
@@ -1346,6 +1348,8 @@ cat >>! $RUNDIR/ocn_in$inststr << EOF
 ! NDEPFILE    : File name (incl. full path) for atmopheric N-deposition data
 ! INIXXX      : Initial condition file for iHAMOCC, where XXX=DIC, ALK, PO4,
 !               OXY, NO3, SIL, D13C, and D14C
+! WITH_DMSPH  : Logical switch to activate DMS calculation as function of pH
+! PI_PH_PATH  : File name (incl. full path) for pH input data
 &BGCNML
```

Comment:
Since this is the full file name the variable name should be "PI_PH_FILE" or maybe "PI_PH_CLIM"

Reply:
I have kept the `PI_PH_FILE` variable name, but specify in the comment that the file name is set to `MONTHLY_PI_PH.nc`, so only the path and not the file name should be specified.